aws/resource_aws_ssm_patch_baseline.go

```diff
@@ -235,7 +241,11 @@ func resourceAwsSsmPatchBaselineCreate(d *schema.ResourceData, meta interface{})
 	}
 
 	if _, ok := d.GetOk("approval_rule"); ok {
-		params.ApprovalRules = expandAwsSsmPatchRuleGroup(d)
+		rules, err := expandAwsSsmPatchRuleGroup(d)
+		if err != nil {
+			return err
+		}
+		params.ApprovalRules = rules
```

Comment:
Inline with the apply-time error removal:

```suggestion
		params.ApprovalRules = expandAwsSsmPatchRuleGroup(d)
```